app/services/getQuestion.js

```diff
@@ -0,0 +1,11 @@
+const config = require('config');
+const request = require('superagent');
+
+const apiUrl = config.get('api.url');
+
+function getQuestion(hearingId, questionId) {
+  return request
+    .get(`${apiUrl}/continuous-online-hearings/${hearingId}/questions/${questionId}`);
```

Comment:
Need to handle an error here too.

Reply:
@davet1985 How do you suggest handling it without next? Just with appInsights?

Reply:
Is `superagent` promise based? If so, I think it would make sense for this function to `resolve` if successful and `reject` if not. What to do with a success or fail can then be handled by the controller, probably just called next with the error in a catch block. What do you reckon?